shaptools/hana.py

```diff
@@ -342,16 +342,26 @@ def get_version(self):
 
     def start(self):
         """
-        Start hana instance
+        Start hana instance.
+
+        Info: 'HDB start' is not used, as this command only operates in the local machine and
+        it does not start machines in other host
         """
-        cmd = 'HDB start'
+        cmd = 'sapcontrol -nr {} -function StartSystem HDB'.format(self.inst)
+        self._run_hana_command(cmd)
+        cmd = 'sapcontrol -nr {} -function WaitforStarted 2700 2'.format(self.inst)
```

Comment:
regarding 2700 2 I'm ok if we don't want `paramaters` but right no they are magic number. we should convert this a constants so an user/we can understand what this number are for

`END_OF_WORLD = '2700'`
`HOW_MANY_ALIENS = 2`